adapters/mqtt-vertx-base/src/main/java/org/eclipse/hono/adapter/mqtt/AbstractVertxBasedMqttProtocolAdapter.java

```diff
@@ -301,17 +302,30 @@ final void handleEndpointConnection(final MqttEndpoint endpoint) {
         }
     }
 
-    private void handleConnectionRequestResult ( final MqttEndpoint endpoint, final AsyncResult<Device> done) {
+    private void handleConnectionRequestResult(final MqttEndpoint endpoint,
+            final AsyncResult<Device> authenticationResult) {
 
-        if ( done.succeeded() ) {
+        if (authenticationResult.succeeded()) {
 
-            endpoint.accept(false);
-            sendConnectedEvent(endpoint.clientIdentifier(), done.result());
+            sendConnectedEvent(endpoint.clientIdentifier(), authenticationResult.result())
+                    .setHandler(result -> {
+                        if (result.succeeded()) {
+                            endpoint.accept(false);
+                        } else {
+                            LOG.info(
```

Comment:
is this worth to be logged at INFO level? If this indicates a problem that needs to be taken care of then it should be WARNING, otherwise it should be DEBUG FMPOV.